src/sentry/integrations/github/client.py

```diff
@@ -219,6 +219,8 @@ def process_error(error: ApiError, extra: Dict[str, str]) -> None:
                 logger.warning("Github has blocked this org. We will not continue.", extra=extra)
                 # Raising the error will be handled at the task level
                 raise error
+            elif txt.startswith("Unable to reach host:"):
+                logger.warning("Unable to reach host at the moment. Continue.", extra=extra)
```

Comment:
Nit: None of the other errors mention `continue` when they continue - we should stay consistent with one of the two approaches.